rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -378,11 +401,24 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
                 counterparty=CPT_COWSWAP,
                 airdrop_identifier=airdrop_identifier,
             ):
+                if claim_has_payment:
+                    event.event_type = HistoryEventType.TRADE
+                    event.event_subtype = HistoryEventSubType.RECEIVE
+                    in_event = event
                 break
```

Comment:
Shouldn't the break be in there? Or we break on outer if and in inner if just change type and remember in event?

```suggestion
                    in_event = event
                    break
```